composer.json

```diff
@@ -34,7 +34,7 @@
   "require-dev": {
     "symfony/security-bundle": "^6",
     "symfony/yaml": "^6",
-    "beberlei/porpaginas": "^1.2",
+    "beberlei/porpaginas": "^1.2 || ^2.0",
```

Comment:
Upgraded this is v2, v2 fixed some deprecation's on php 8.1, I couldn't find any breaking change that could affect this pkg